jaqpotpy/doa/tests/test_doa.py

```diff
@@ -90,4 +90,35 @@ def test_mean_var(self):
         assert len(calc) == len(mol)
         assert calc[0]['IN']==True, f"Expected calc[0]['IN'] == True, got {calc[0]['IN']} != True"
         assert calc[1]['IN']==False, f"Expected calc[0]['IN'] == False, got {calc[1]['IN']} != False"
-        assert np.allclose(diag, [1.31511044e+01, 6.69162726e-01, 5.37187947e-03], atol= 1e-5), f"Expected diag == [1.31511044e+01, 6.69162726e-01, 5.37187947e-03], got diag != {diag}"
\ No newline at end of file
+        assert np.allclose(diag, [1.31511044e+01, 6.69162726e-01, 5.37187947e-03], atol= 1e-5), f"Expected diag == [1.31511044e+01, 6.69162726e-01, 5.37187947e-03], got diag != {diag}"
```

Comment:
Rename this function to test_meanvar since you are using the MeanVar doa here. Also please rename this "IN" method because it's not pythonic at all as we discussed